### PR TITLE
Lyrics Sync in Audio Player

### PR DESCRIPTION
**Pull request by shea786**
Display the lyrics correctly on the audio player

### CODE REVIEW

File: web/src/components/audio-player/AudioPlayer.vue [hunk 1]
```diff
@@ -792,17 +799,21 @@ $duration: 680ms;
   bottom: 0;
   right: 0;
   white-space: nowrap;
-  z-index: 500;
-  box-shadow: 0 -2px 8px 4px rgba(0, 0, 0, 0.16);
+  z-index: 10;
```
Comment (szainmehdi) [inline web/src/components/audio-player/AudioPlayer.vue:802]: What was the reasoning for this change?

File: web/src/components/audio-player/AudioPlayer.vue [hunk 2]
```diff
@@ -878,80 +888,74 @@ $duration: 680ms;
 }
 
 @media #{map-get($display-breakpoints, 'md-and-down')} {
-  .audio-player {
-    z-index: 3 !important;
-  }
+    .audio-player {
+      z-index: 3 !important;
+    }
```
Comment (szainmehdi) [inline web/src/components/audio-player/AudioPlayer.vue:893]: I think this indentation is wrong

File: web/src/components/audio-player/AudioPlayer.vue [hunk 3]
```diff
@@ -775,6 +757,31 @@ export default class AudioPlayer extends Vue {
...
+          return;
+        }
+        this.vibrantBackgroundColor = swatch.getHex();
+        this.vibrantTextColor = swatch.getBodyTextColor();
+      });
+  }
+
+  scrollToCurrentLyricsGroup(id) {
+    if (id === null) {
+      window.scrollTo(0, 0);
```
Comment (szainmehdi) [inline web/src/components/audio-player/AudioPlayer.vue:776]: I don't understand what this is doing.

File: web/src/components/audio-player/AudioPlayer.vue (hunk 1 above)
Comment (shea786) [reply web/src/components/audio-player/AudioPlayer.vue:786]: I don't remember changing that

File: web/src/components/audio-player/AudioPlayer.vue (hunk 3 above)
Comment (shea786) [reply web/src/components/audio-player/AudioPlayer.vue]: So currently we have a track that is playing with the new lyrics format

Now when the next track starts to play and the next track is the old format, then it doesnt scroll to the top automatically, so this line is scrolling to the top of the component if we do not have any ID

**Pull request merged**